### PR TITLE
fix dashboard "Discord Requests" to "Request Review"

### PR DESCRIPTION
**Pull request by AldrikRamos**
xx

### CODE REVIEW

File: website-html/admin-community-update.html [hunk 1]
```diff
@@ -26,7 +26,7 @@
 							|
 							<a href="admin-discord-requests.html" class="no-style more" class="no-style more">Discord Requests</a>
 							|
-							<a href="admin-discord-review.html" class="no-style more">Manage Discord</a>
```
Comment (skyler1ackerman) [inline website-html/admin-community-update.html:29]: Are we changing this as well? I thought it was just the Discord Requests thing

**Review by skyler1ackerman — CHANGES_REQUESTED** [review]

File: website-html/admin-community-update.html (hunk 1 above)
Comment (AldrikRamos) [reply website-html/admin-community-update.html:29]: good catch, sloppy work here

**Comment by AldrikRamos** [comment]
fixed

**Review by skyler1ackerman — APPROVED** [review]

**Comment by skyler1ackerman** [comment]
LGTM

**Comment by AldrikRamos** [comment]
extra fix, button text changed for community update to keep it same as "ACCEPT" "REJECT" in other review requests and the artifacts

**Comment by skyler1ackerman** [comment]
LGTM Still

**Pull request merged**